src/app/portfolio/page.tsx

```
@@ -1,7 +1,52 @@
-export default function Page() {
+import { dehydrate, HydrationBoundary } from "@tanstack/react-query";
+import ProjectList from "./_components/ProjectList";
+import WriteLink from "./_components/WriteLink";
+import getQueryClient from "@/components/QueryClient";
+import { getProjectList } from "@/services/Project.api";
+import { getUser } from "@/services/user.api";
+import { User } from "@/types/AuthType";
+import { Project } from "@/types/PortfolioType";
+
+interface ProjectList {
```

Biome

[error] Shouldn't redeclare 'ProjectList'. Consider to delete it or rename it.

'ProjectList' is defined here:


(lint/suspicious/noRedeclare)